pkgs/development/python-modules/vosk/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, srt
+, requests
+, tqdm
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "vosk";
+  version = "0.3.45";
+  format = "wheel";
+
+  src = fetchPypi {
+    inherit pname version format;
+    python = "py3";
+    dist = "py3";
+    platform = "manylinux_2_12_x86_64.manylinux2010_x86_64";
+
+    hash = "sha256-JeAlCTxDmdcnj1Q1aO2MxUYKw6S/SMI2c6zh4l0mYZ8=";
+  };
```

Comment:
Package has a github repository
https://github.com/alphacep/vosk-api/tree/master/python

Using `fetchFromGitHub` will also download test files, which could be run after install phase.